### PR TITLE
Tarask/fix/testkube helm release

### PR DESCRIPTION
**Pull request by fog1985**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: charts/testkube/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: v2
+name: testkube
+description: A Helm chart for testkube. 
+type: application
+version: 0.5.84
+dependencies:
+- name: testkube-operator
+  version: 0.5.8
+  #repository: https://kubeshop.github.io/helm-charts
+  repository: "file://../testkube-operator"
```
[the remark this answers is not in the document]
Comment (exu) [reply charts/testkube/Chart.yaml:10]: 👍🏻

File: charts/testkube/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+apiVersion: v2
+name: testkube
+description: A Helm chart for testkube. 
+type: application
+version: 0.5.84
+dependencies:
+- name: testkube-operator
+  version: 0.5.8
+  #repository: https://kubeshop.github.io/helm-charts
```
Comment (nicufk) [inline charts/testkube/Chart.yaml:9]: Local repos.

File: charts/testkube/Chart.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fog1985) [reply charts/testkube/Chart.yaml:10]: It just uses helm charts (dependencies) from the folders in this repo. No need to request the same via remotes...

**Review by exu — APPROVED** [review]
LGTM

**Pull request merged**